src/legacy-html-from-paste.js

```diff
@@ -24,7 +24,14 @@ function getHtmlUsingHiddenElement (hiddenElement) {
   return hiddenElement.innerHTML
 }
 
+function haveClipboardAccess () {
```

Comment:
I think it'd be useful to have a comment explain what this does and how it works - since it's pretty whacky.

Reply:
Added the comment from the commit message. Let me know if that makes sense.